ts/components/wallet/CreditCardComponent.tsx

```diff
@@ -169,7 +170,10 @@ export default class CreditCardComponent extends React.Component<Props> {
               style={CreditCardStyle.rowStyle}
               size={2}
               {...{
-                onPress: (): boolean => navigate("")
+                onPress: (): boolean =>
+                  navigate(ROUTES.WALLET_CARD_TRANSACTIONS, {
```

Comment:
+ TODO for later removal?

Reply:
Can you elaborate on this comment? This is where the user taps on a CC and navigates to the list of transactions carried out with the selected CC -- any reason why this should be removed later?

Reply:
sorry I thought it was "the shortcut" for reaching the screen, never mind